src/Native/build.proj

```diff
@@ -90,6 +90,8 @@
                           RelativePath="Microsoft.ML\runtimes\$(PackageRid)\native" />
       <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)FactorizationMachineNative$(NativeLibExtension)"
                           RelativePath="Microsoft.ML\runtimes\$(PackageRid)\native" />
+      <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)SymSgdNative$(NativeLibExtension)"
+                          RelativePath="Microsoft.Hal\runtimes\$(PackageRid)\native" />
```

Comment:
The real name here is: `Microsoft.ML.HalLearners`

Reply:
It was on accident.

---
In reply to: [206970539](https://github.com/dotnet/machinelearning/pull/624#discussion_r206970539) [](ancestors = 206970539)